en/02_Developer_Guides/15_Customising_the_Admin_Interface/02_CMS_Architecture.md

```diff
@@ -338,28 +298,33 @@ can be used to pass state between route handlers and inspect the current
 history state. The `next` function invokes the next matching route. If `next`
 is called when there is no 'next' route, a page refresh will occur.
 
-### React router CMS sections
+#### React router CMS sections
 
 Similarly to page.js powered routing, the main point of registration for react routing
 sections is the `LeftAndMain::getClientConfig()` overridden method, which controls the main
-routing mechanism for this section. However, there are two major differences:
+routing mechanism for this section. However, there are three major differences:
+
+Firstly, where the page.js router uses the full path (e.g. 'admin/pages') as its route,
+react-router uses relatively paths with nested routes. The main route is, by default, 'admin',
+which means the 'admin/pages' path would declare its route as 'pages'. This is added against the
+`reactRoutePath` key in the array returned from `LeftAndMain::getClientConfig()`.
```

Comment:
react-router now uses relative paths. The `reactRoutePath` was added to accomodate this.